GetJobber/atlantis
# feat(components): Atlantis Component For Gallery

## Motivations

I added a new component `Gallery` to atlantis, which displays a collection of thumbnails to the viewer

## Changes


* New component added
* Modified FormatFile so that it has a tab index, so that it can be focusable
* Modified FormatFile css so that it has a new class `thumbnailParent`, to get descendant combinator interactive selectors to work (e.g., hover)

### Added

- `Gallery` component

### Changed

- `FormatFile`:
    - Now tab focusable
    - New css classes allowing for focus and hover

## Testing

- Run atlantis on this PR locally
- Go to the Gallery component. Look at the test cases and/or even play around with the Playground to test your own changes
- Ensure that everything matches the figma mockup for Gallery --> https://www.figma.com/file/b7LY6hVMwFOj5maidxPTMB/Design-System-Contribution-%5B-Thumbnail-%2B-Gallery-%5D?node-id=1420%3A19639

## References

Figma Mockup --> https://www.figma.com/file/b7LY6hVMwFOj5maidxPTMB/Design-System-Contribution-%5B-Thumbnail-%2B-Gallery-%5D?node-id=1420%3A19639

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
